src/SiteEnrollment/NewSiteInfo.js

```diff
@@ -445,7 +507,10 @@ export const NewSiteInfo = ({
                   <Button
                     variant="contained"
                     color="primary"
-                    onClick={handleUpdateNewSite}
+                    onClick={() => {
+                      console.log('yooooooooooooooooo');
```

Comment:
please remove this debug clg

Reply:
Yes, I'll do that. I apologize for the bug.